core/lib/spree/testing_support/common_rake.rb

```diff
@@ -22,9 +22,11 @@
     sh "bin/rails db:drop db:create db:migrate VERBOSE=false RAILS_ENV=test"
 
     begin
-      require "generators/#{ENV['LIB_NAME']}/install/install_generator"
+      generator_namespace = "#{ENV['LIB_NAMESPACE']&.underscore || ENV['LIB_NAME']}"
+
+      require "generators/#{generator_namespace}/install/install_generator"
```

Comment:
Why do we need this require if the generator is ran with sh?

Reply:
@kennyadsl Good question! Let me test if `common:test_app` will run successfully if there is no extension generator. TODO

Reply:
@kennyadsl It's a bit hacky, but the `begin` block functions as a conditional block:

* If `common:test_app` is ran in a Solidus extension that has an installation generator, the rake task would print out "Running extension installation generator..." and run the generator. 
* On the other hand, if `common:test_app` is ran in a Solidus extension that does not have an installation generator, the rake task would throw a load error when it tries to require the generator, and the task would skip the extension installation process.

I wasn't able to find a cleaner way to check if a generator exists, but perhaps we can reorganize the code to make the intent clearer, like this:

```ruby
    extension_installation_generator_exists =
      begin
        require "generators/#{generator_namespace}/install/install_generator"

        true
      rescue LoadError
        false
      end

    if extension_installation_generator_exists
      puts 'Running extension installation generator...'
      sh "bin/rails generate #{generator_namespace}:install --auto-run-migrations"
    else
      puts 'No extension installation generator found...'
    end
```

I intend to go ahead with this refactor. Let me know if you have additional thoughts!

Reply:
I think it makes total sense. Please go ahead with this!